obspy/io/gse2/tests/data/boa___00_07a.gse

```diff
@@ -140,4 +141,4 @@ DAT2
 9 9 7 7 6 5 8 10 6 2 3 6 7 7 4 1 -3 -2 2 3 0 -5 -7 -5 1 2 -4 -9 -8 -4 1 2 1 -6 -10 -6 1 1 0 -2 -4 -3 2 3 1 0 -2 -4 -6 -6 -7 -8 -7 -5
 -5 -8 -11 -12 -11 -9 -9 -10 -13 -15 -15 -12 -7 -7 -10 -12 -13 -10 -5 -4 -10 -14 -17 -18 -13 -12 -14 -15 -16 -16 -13 -11 -14 -15 -14
 -13 -10 -6 -6 -6 -3 0 1 1 0 -4 -4 0 0 -2 -5 -8 -8 -4 -3 -3 -3 -5 -4 2 9 10 6 4 2 4 10 12 8 4 4 5 7 10
-CHK2     5140
+CHK2     -5140
```

Comment:
Uhm.. how come this file had a wrong checksum? Was the checksum wrong in the file we got from the mail on the users mailing list? Funny, then there's another program out there mixing up the sign of the checksum, I guess..?